dist/tools/cpy2remed/cpy2remed.sh

```diff
@@ -1,4 +1,10 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/sh
```

After applying @benpicco's comment, this should be POSIX shell compliant, shouldn't it? If not, the CI's `static-test` runs shellcheck that will tell you what is in violation with the POSIX standard.

It would be quite nice to not have this dependend on bash. E.g. Mac OS uses zsh by default, debian uses dash by default, Alpine uses ash by default, etc. But they all accept POSIX compliant shell scripts.